koku/masu/test/util/azure/test_azure_post_processor.py

```diff
@@ -99,7 +99,7 @@ def test_finalize_post_processing(self):
         self.post_processor.finalize_post_processing()
 
         with schema_context(self.schema):
-            tag_key_count = AzureEnabledTagKeys.objects.filter(key__in=expected_tag_keys).count()
+            tag_key_count = EnabledTagKeys.objects.filter(provider_type="Azure", key__in=expected_tag_keys).count()
```

Comment:
We need to be consistent in how we filter. In the aws file, we used `Provider.PROVIDER_AWS`, but here and GCP/OCI, we're using strings. Pick one or the other.

Reply:
Yeah, That is a by product of having three different engineers work on the same branch 😂 

Ill get these these cleaned up.